graph/builder.go

```diff
@@ -681,7 +681,7 @@ func (b *Builder) handleNetworkUpdate(vb *ValidationBarrier,
 			update.err <- err
 
 		case IsError(err, ErrParentValidationFailed):
-			update.err <- newErrf(ErrIgnored, err.Error())
+			update.err <- NewErrf(ErrIgnored, err.Error()) //nolint
```

Comment:
hmm weird we got this new failure but not in the original pr

Reply:
Probably some linter rule changes not in this branch, can likely ignore.